kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md

```diff
@@ -73,7 +73,20 @@ In the comments next to each cell, we've marked which component of the YellowPap
             </callState>
           </evm>
         </ethereum>
+```
+
+For symbolic execution, we create an additional cell, to capture the
+side-effects of impure hooks.
+
+```symbolic
+        <worldState> .WorldState </worldState>
+
+    syntax WorldState ::= ".WorldState" [function, total]
+    rule <k> WS:WorldState => .K ... </k>
+         <worldState> _ => WS </worldState>
+```
```

Comment:
I'm unsure about this naming here. Previously, we didn't use the `pushWorldState`, `popWorldState`, and the `dropWorldState` from the original semantics, so this variable name was ok. But now, we have reverted this change, and we're using this structure here. As they are different things, they should have different names, right?

Reply:
If you've got a better idea for a name, let me know and I can change it. Otherwise we can change it in a later PR.

Reply:
Sorry, I don't follow? What do you mean? They both seem to include the network state, and transation outputs if thats what you mean.